book/pend.html

```diff
@@ -709,13 +709,14 @@ <h1>The Simple Pendulum</h1>
 
     <exercise><h1>Pendulum with Vibrating Base</h1>
 
-      Consider an actuated pendulum whose base (pivot of the rod) is forced to oscillate horizontally according to the harmonic law $h \sin(\omega t)$, where $h$ denotes the amplitude of the oscillation and $\omega$ the frequency.  The equation of motion for this system is $$m l^2 \ddot \theta + m g l \sin \theta = m l \omega^2 h \sin (\omega t) \cos \theta + u.$$  (If this equation seems obscure to you, try to derive it using the <a href="multibody.html">Lagrangian approach</a>; but be careful, the kinetic energy $T(\theta, \dot \theta, t)$ of this system depends explicitly on time.)  Our goal is to design a time-dependent control law $u = \pi (\theta, \dot \theta, t)$ that makes the pendulum spin at constant velocity $\dot \theta = 1$.
-
       <ol type="a">
 
-        <li>Identify a desired closed-loop dynamics $\ddot \theta = f(\dot \theta)$, whose unique equilibrium point is stable and is located in $\dot \theta = 1$.</li>
+        <li>Identify the family of dynamics functions $\ddot \theta = f(\dot \theta)$ whose unique equilibrium point is stable and is located at $\dot \theta = 1$.</li>
```

Comment:
i actually don't like this problem... even before your changes.  \dot\theta = 1 is _never_ an equilibrium point?

Reply:
In variables ($\theta, \dot\theta$), the differential equation is  $\dot {(\theta, \dot\theta)} = (\dot\theta, f(\dot\theta) )$, and the equilibrium for this system requires $\dot\theta = 0$. 

But in just one variable $\dot\theta$, differentiable equation is $\dot{(\dot\theta)} = f(\dot\theta)$, and there exists a function $f(\cdot)$ for which $\dot\theta=1$ is an equilibrium point.

I can totally see how this may be confusing though.

Reply:
I still like the feedback cancellation component of this problem, "look at the cool things you can do when you're fully actuated!"